benchmarking/prototype_parameterized_template.py

```
@@ -0,0 +1,613 @@
+        )
+        values = [
+            np.asarray(item["defaults"][variable.name], dtype=variable.type.dtype)
+            for variable in item["explicit"]
+        ]
+        compiled_items.append((item, compiled, values))
+        construction_times.append(elapsed)
+        graph_nodes.append(len(compiled.maker.fgraph.apply_nodes))
+
+    results = []
+    first_times = []
+    for item, compiled, values in compiled_items:
```

Ruff (0.16.0)

[warning] Loop control variable `item` not used within loop body

(B007)